.devpanel/init.sh

```diff
@@ -15,31 +15,16 @@
 # For GNU Affero General Public License see <https://www.gnu.org/licenses/>.
 # ----------------------------------------------------------------------
 
-STATIC_FILES_PATH="$WEB_ROOT/sites/default/files"
-SETTINGS_FILES_PATH="$WEB_ROOT/sites/default/settings.php"
 #== Clone source code
 if [ ! -d "$APP_ROOT/starshot-prototype" ]; then
-  git clone https://github.com/phenaproxima/starshot-prototype.git starshot-prototype
+  git clone -b install-fails-with-database-url-argument https://github.com/darrenoh/starshot-prototype.git
 fi
 
-#== Setup settings.php file
-sudo cp $APP_ROOT/.devpanel/drupal-settings.php $SETTINGS_FILES_PATH
-
```

Comment:
You removed the settings.php file from Devpanel, which means you cannot connect to the database using settings.php from Starshot.

Reply:
The database settings are being passed to the Starshot install script on line 29.